src/api/bind-contract-function.js

```diff
@@ -23,9 +23,8 @@ function bindContractFunction(functionAbi) {
       }
       if (isFunction(params[params.length - 1])) callback = params.pop();
       if (!isFunction(callback)) return ethrpc.callContractFunction(payload);
-      return ethrpc.callContractFunction(payload, function (response) {
-        if (response == null) return callback("No response");
-        if (response.error != null) return callback(response.error);
+      return ethrpc.callContractFunction(payload, function (err, response) {
+        if (err) return callback(err);
```

Comment:
Should this also have a check to ensure `response` is not null?

Reply:
Hm, no I don't think that is necessary -- if response is null, ethrpc should pass back an error.

Reply:
Actually it appears you're right -- ethrpc is just checking for `undefined`, not `null` results.  So there is an extra check needed here.  Nice catch!